src/components/BookForm.js

```diff
@@ -19,10 +19,13 @@ function BookForm() {
   }
 
   return (
-    <form onSubmit={handleSubmit}>
-      <input type="text" placeholder="Title" value={title} required onChange={(ev) => setTitle(ev.target.value)} />
-      <input type="text" placeholder="Author" value={author} required onChange={(ev) => setAuthor(ev.target.value)} />
-      <button type="submit">Add Book</button>
+    <form className="form" onSubmit={handleSubmit} action="">
+      <h2>Add new book</h2>
+      <div className="container">
+        <input className="title" type="text" placeholder="Title" value={title} required onChange={(ev) => setTitle(ev.target.value)} />
+        <input className="author" type="text" placeholder="Author" value={author} required onChange={(ev) => setAuthor(ev.target.value)} />
+        <button className="form-btn" type="submit">Add Book</button>
```

Comment:
- [ ] Great job working on this milestone. You have an aligment issue. As per the Zeplin design in the screenshot below, ADD BOOK button should align with the section above having the UPDATE PROGRESS button. This is not the case with your design.
![Screen Shot 2023-04-15 at 3 44 32 AM](https://user-images.githubusercontent.com/62849872/232209340-061fbaa0-dfa0-4127-ba5d-8cd571b3500c.png)